skietbaan_fe/src/components/Posts.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux';
+import { fetchPosts } from '../actions/postActions'
+
+class Posts extends Component {
+
+  // a constructor that maintanes the state of the component
+ /**  constructor(props ){
+    super(props);
+    this.state = {
+      posts: []
+    }
+  } */
+componentWillMount(){
+  this.props.fetchPosts();
+
+}
+
+componentWillReceiveProps(nextProps){
+  if(nextProps.newPost){
+      this.props.posts.unshift(nextProps.newPost);
+  }
+}
+
+//the method to render the 
```

Comment:
the what?

Reply:
commented out code should never make it to the other side of life. ..

Reply:
This is quite a misleading comment @CeeyarRbt because:
- Its not commented out code its actually a comment.
- There are times when ghost code is pushed with a *TODO: reason*

Reply:
You can optimize your if statement in one line:
(nextProps.newPost) ? this.props.posts.unshift(nextProps.newPost) : '';

Always think of the else statement just in case things go south.

Reply:
> You can optimize your if statement in one line:
> (nextProps.newPost) ? this.props.posts.unshift(nextProps.newPost) : '';
> 
> Always think of the else statement just in case things go south.

in this case I disagree. You do not always need an else statement 
Also using `(nextProps.newPost) ? this.props.posts.unshift(nextProps.newPost) : '';` is not valid. Why an empty string at the end? you aren't assigned the result to anything. So I say leave the if as is currently